api-get-object.go

```diff
@@ -26,6 +26,30 @@ import (
 	"time"
 )
 
+// GetSecuredObject returns a readable object
```

Comment:
Talk about what the `GetSecuredObject()` does i.e provides a `deciphered object` data - deciphered using the encryption materials.